packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -125,6 +125,14 @@ handler.addPR = async function addPR(wp: WatchPR, url: string) {
  */
 function handler(app: Application) {
   app.on(['schedule.repository'], async context => {
+    const rateLimit = (await context.github.rateLimit.get()).data.resources.core
```

Comment:
When this happens - do we think it's a regular "log" event, or an "error"?  Thinking out loud - I think this is an error.  It means we've built things in a way that it can't live within the API rate limits we're given and it's setting us up for trouble.  Not sure what the answer is to manage it moving forward, but I don't want to treat this as business as usual when it happens.

Reply:
Great point. I made some changes to try to lower the calls, but I agree. Created tracking issue #918 for etags.